test/e2e/e2e.go

```
@@ -0,0 +1,630 @@
+
+	configClient, err := configclient.NewForConfig(config)
+	o.Expect(err).NotTo(o.HaveOccurred(), "failed to create config client")
+
+	ctx := context.TODO()
+
+	// Wait for operator to report healthy status
+	var clusterOperator *configv1.ClusterOperator
+	err = wait.PollUntilContextTimeout(ctx, pollInterval, pollTimeout, true, func(ctx context.Context) (bool, error) {
+		co, getErr := configClient.ConfigV1().ClusterOperators().Get(ctx, "config-operator", metav1.GetOptions{})
+		if getErr != nil {
+			return false, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 140) but it returns nil

(nilerr)